src/pages/[section]/[page]/[...tab].astro

```diff
@@ -0,0 +1,107 @@
+---
+import { getCollection, render } from 'astro:content'
+import { Title, PageSection, Content as PFContent } from '@patternfly/react-core'
+import MainLayout from '../../../layouts/Main.astro'
+import { content } from "../../../content"
+import { kebabCase } from 'change-case'
+import { componentTabs, tabNames } from '../../../globals';
+
+export async function getStaticPaths() {
+  const collections = await Promise.all(content.map(async (entry) => await getCollection(entry.name as 'textContent')))
+
+  const flatCol =  collections.flat().map((entry) => {
+    // Build tabs dictionary
+    let tab = entry.data.tab;
+    if(tab === 'react' || tab === 'html') { // if tab is default repo value, check for demos/deprecated
+      if(entry.id.includes('demos')) {
+        tab = `${tab}-demos`;
+      } else if (entry.id.includes('deprecated')) {
+        tab = `${tab}-deprecated`;
+      }
+    }
+
+    // if no dictionary entry exists, and tab data exists
+    if(componentTabs[entry.data.id] === undefined && tab) { 
+      componentTabs[entry.data.id] = [tab];
+    // if dictionary entry & tab data exists, and entry does not include tab
+    } else if (componentTabs[entry.data.id] && tab && !componentTabs[entry.data.id].includes(tab)) { 
+      componentTabs[entry.data.id] = [...componentTabs[entry.data.id], tab];
+    }
+
+    return {
+      params: { page: kebabCase(entry.data.id), section: entry.data.section, tab },
+      props: { entry, title: entry.data.title, component: entry.data.id,  package: entry.data.package },
+    }
+  })
+
+  const defaultOrder = 50;
+  const sourceOrder: any = {
+    react: 1,
+    'react-next': 1.1,
+    'react-demos': 2,
+    'react-deprecated': 2.1,
+    html: 3,
+    'html-demos': 4,
+    'design-guidelines': 99,
+    'accessibility': 100,
+    'upgrade-guide': 101,
+    'release-notes': 102,
+  };
```

Comment:
I'd be fine with this being in a followup if you'd prefer that, but I feel like this kind of thing should ideally come from the config file being passed into the docs-core.